libc/docs/overlay_mode.rst

```diff
@@ -39,7 +39,7 @@ the CMake configure step as follows:
   $> cd llvm-project  # The llvm-project checkout
   $> mkdir build
   $> cd build
-  $> cmake ../llvm -G Ninja -DLLVM_ENABLE_PROJECTS="libc"  \
+  $> cmake ../llvm -G Ninja -DLLVM_ENABLE_RUNTIMES="libc"  \
```

Comment:
As a side note, `LLVM_ENABLE_PROJECTS` should work but the cmake is unhappy about it, generating no targets of libc for some reason. I think it is generally better to use `LLVM_ENABLE_RUNTIMES` after all.